docker-compose.yml

```
@@ -19,7 +19,7 @@ services:
   db:
     expose:
       - "5432"
-    image: postgres:10-alpine
+    image: postgres:13
```

To keep using alpine, we should probably use 13-alpine
```suggestion
    image: postgres:13-alpine
```

---

I changed from alpine because it was suggested in #54. Docker compose is not an official way to run Unleash, more for testing purposes. 

Both works on my machine though!